fix(maven-use): always set version (#1139)
canonical/craft-parts: docs/reference/changelog.rst

  For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.

X.Y.Z (2025-MM-DD)
------------------

Bug fixes:

> - With the maven-use plugin, do not raise errors if versions are not specified
>   in a project's :file:`pom.xml`.

Reviewer: ```suggestion
- With the maven-use plugin, don't raise errors if dependency versions aren't specified
  in the software's :file:`pom.xml` file.
```
